AppSettingsManagerBff/AppSettingsManagerBff.Domain/ApiRepositories/IUserRepository.cs

```diff
@@ -1,11 +1,14 @@
-using AppSettingsManagerBff.Model.Api;
+using AppSettingsManagerBff.Model;
+using AppSettingsManagerBff.Model.Requests;
 
 namespace AppSettingsManagerBff.Domain.ApiRepositories;
 
-public interface IUserRepository
+public interface ISettingsRepository
 {
-    Task<ApiBaseUser> GetUser(string userId);
-    Task<ApiBaseUser> CreateUser(string userId, string password, string email);
-    Task<ApiBaseUser> UpdateUser(string userId, string newPassword);
-    Task<ApiBaseUser> DeleteUser(string userId);
-}
+    Task<SettingGroup> GetSettingGroup(string userId, string password, string settingGroupId);
+    Task<IEnumerable<SettingGroup>> GetSettingGroupsForUser(string userId, string password);
+    Task<SettingGroup> CreateSettingGroup(CreateSettingRequest request);
+    Task<SettingGroup> UpdateSetting(CreateSettingRequest request);
+    Task<SettingGroup> ChangeTargetSettingVersion(UpdateTargetSettingRequest request);
+    Task<SettingGroup> DeleteSettingGroup(string userId, string password, string settingGroupId);
```

Comment:
It looks like this got copied over the user repository interface somehow. Go back in your commits and find a good version. Or just recreate it based on the HttpUserRepository

Reply:
I am wondering the same thing. Looks like you accidentally wrote code for the ISettingsRepository under the IUserRepository.cs file. It seems like an obvious mistake, but should be easy to fix.

Reply:
Oops, my bad, I copied over the stuff from ISettingRepository and forgot to change it. Going to fix it now.